docs/user_guide/datasets/index.rst

```diff
@@ -0,0 +1,13 @@
+.. _datasets:
```

Comment:
If you create a PR with just this new file and not even the adult page it'll probably unblock @britneyting since she can base her changes on it. Definitely in the spirit of #LutzsLaw 😉

Reply:
Fair point. I think this PR is almost ready to merge though? But I can do that if needed.